infra-templates/k8s/53/rancher-compose.yml

```diff
@@ -1,6 +1,6 @@
 .catalog:
   name: Kubernetes
-  version: v1.11.2-rancher1-1
+  version: v1.11.2-rancher1-2
```

Comment:
@deniseschannon as we only bump up the image for lb controller w/o changing the k8s base image, wanted to make sure that the version semantics is followed correctly